client/src/components/MinMaxSelect.tsx

```diff
@@ -35,6 +50,25 @@ function MinMaxSelect(props: {
 
   const hasCustomInputs = isFinite(customRange.min) || isFinite(customRange.max);
 
+  React.useEffect(() => {
+    if (!isOpen) {
```

Comment:
for code clarity, i'd change this to a switch statement.
`switch (!isOpen) {
    case (defaultSelections.type === "custom"):
        ...
        break;
    case (defaultSelections.type === "presets"):
        ...
        break;
    default:
        ...
}
`

Reply:
that's my only note for this PR before approving!

Reply:
good idea! much nicer. just updated